src/jit/codegenarm64.cpp

```diff
@@ -1454,7 +1454,17 @@ void CodeGen::genSetRegToConst(regNumber targetReg, var_types targetType, GenTre
             }
             else
             {
-                genSetRegToIcon(targetReg, cnsVal, targetType);
+                if (varTypeIsSIMD(targetType))
```

Comment:
xarch has `genSIMDZero`; shouldn't all simd zeroing go through the equivalent on arm64?

Reply:
This may have been written before I had enabled `genSIMDZero`  will check if this is hit when I get back to the office.

Reply:
Two tests are still hitting this, even with `SIMDIntrinsicGetZero` & `SIMDIntrinsicInit` implemented.  I will have to look into why the nodes are created.

```
JIT/Regression/JitBlue/GitHub_12950/GitHub_12950/GitHub_12950.sh
JIT/SIMD/Ldfld_ro/Ldfld_ro.sh
```

Reply:
I don't think this is the correct fix. I *think* the only place this should be encountered is in a `GT_STORE_LCL_VAR`, which corresponds to an struct init (not a `SIMDIntrinsicInit`). It is handled in xarch here: https://github.com/dotnet/coreclr/blob/master/src/jit/codegenxarch.cpp#L4494-L4501

Reply:
Thanks @CarolEidt